style.css

```diff
@@ -106,8 +110,118 @@
     .btn{
         text-align: left;
```

Comment:
Social icons are not centered

![Screenshot 2023-03-13 at 2 47 37 PM](https://user-images.githubusercontent.com/38748298/224658258-7813e2ee-089f-475f-ace0-cda97e89ac4a.png)